src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -50,8 +58,9 @@ class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter, allowHea
     * @return Passes through the `payload` argument
     */
   private def measureFrameBytesSent(eventName: String, payload: String): Unit = {
-    val overhead: Long = 16L + eventName.length
-    bytesWrittenMetric.increment(payload.length + overhead)
+    val overhead: Long = 16L + eventName.length.toLong
```

Comment:
ai: magic number 16... I realize it was there before... it would be great to have a variable name that represents the reason why 16 is so magical

Reply:
I believe `sseFrameOverhead` would be appropriate. @wavesoft, what do you think?